spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusMetricsExportAutoConfigurationTests.java

```diff
@@ -287,7 +295,7 @@ PrometheusScrapeEndpoint customEndpoint(CollectorRegistry collectorRegistry) {
 
 	@Configuration(proxyBeanMethods = false)
 	@Import(BaseConfiguration.class)
-	static class ExemplarsConfiguration {
+	static class SpanConfiguration {
```

Comment:
Would it make sense calling this `DefaultExemplarSamplerConfiguration` and the other one `CustomExemplarSamplerConfiguration`?

Reply:
Agree with you even if imo `DefaultExemplarSamplerConfiguration` expresses more what the configuration will be used for instead of what it actually configures, but the latter seems less important in this test context.
Agree also the use of terminology `Custom*` is harmonious with the style of this test class.
Changed